### ISSUE TITLE
Decrease the time a cow needs to spawn (cheese production)

### ISSUE THREAD

**Issue by ByBurton** [Sh0wdown/UnofficialCrusaderPatch#149]
Cheese production is pretty bad right now. How about decreasing the time a cow needs to spawn back?

(The same could be interesting with stuff like pitch production. Pitch is not worth a lot, so decreasing the time it takes to produce could be useful.)

**Comment by Evrey** [comment]
Or increase the cheese output per cow instead. Decreasing the cow spawn rate also affects tanners in a possibly unwanted way.

**Comment by ByBurton** [comment]
> Decreasing the cow spawn time also affects tanners in a possibly unwanted way.

I mean, compare it to how fast oxes spawn. Damn female hating developers :D

> Or increase the cheese output per cow instead.

Why not both?

**Comment by irgendein-alf** [comment]
Afaik 10 apple farms == 14 cheese farms (both of them at equal distances from the food store) (without producing any tanner stuff), assuming there' 3/3 cows around at time.start=0
so an decrease of cheese producing-time or an increase of the cheese output would be fair, considering 
a) cows need to spawn, 
b) you will get +1 happiness if you both use apple and "cheese farms".
this would still mean that apple farms are a very small minimum better than cheese farms but this change will give the player the option to utilize both.
=>
either increase cheese output by 40% or decrease production time of cheese by 40% (or change both values by 18,32% (1,1832^2=1,39996...))

**Comment by Heroesflorian** [comment]
> I mean, compare it to how fast oxes spawn. Damn female hating developers :D

> Why not both?

Well, cows have nothing to (actively) do beyond eating grass, though. Oxen on the other hand have hard work to do that requires them to move across half of the map... and cannot be used by tanners. 

Besides, if cows respawn nearly immediately, then what is the point of having that mechanic at all? I think dairy farms were intended as some sort of in-between between the simplicity and quick-start capabilities of apple orchards, and complex, long-running bread production chains. 
As such, having them output a fair amount of cheese per cow / milking (on top of being useful for leather armour), but still be a bit more time consuming to get going initially seems like a fair tradeoff. 

There is no point in making them "OP" - quick to start, lots of food production per farm, usability for leather production, all in one...

**Comment by ByBurton** [comment]
Who is talking about near immediately?
If you play on speed 60, which is my normal speed, you see that it takes forever. It takes as long as the farmer needs to milk the cow.

**Comment by ByBurton** [comment]
I am talking about a cut of maybe 20% of the respawn time.
It would buff catapults as well (throwing cows).

**Comment by Heroesflorian** [comment]
> It would buff catapults as well (throwing cows).

Makes me shiver when thinking of multiplayer, and will not matter for many AI characters who do not use cows (or use cows but not for catapults), and generally speaking catapults might be the most useful siege engine for offense (for AI) already. Thus, I am not sure this is an argument pro faster cow respawning. 
To be honest though, if we can agree on an increased production after the cows spawned, and considering the suggested buffs for plague clouds, I think a cow spawning rate buff on top of that is something that would require a bit of testing. 

Plus, if in the future more siege engines can be used for defense (and more AIVs incorporate those), then the entire siege engine balancing and number of siege engines per attack will have to be modified as well, or else the catapult cow throwing will be useless either way, as the catapult will die before it can throw cows, or at least before there are gaps in the defenses to have proper targets for thrown cows in the first place.

**Comment by ByBurton** [comment]
If you all think the consequences are too many, or too significant, then can we at least slightly increase the speed a farmer needs to milk a cow?

**Comment by ByBurton** [comment]
Yes. I think cheese should be just as good as apples, or I'd rather like cheese production to be more effective.
They cost double the amount of an apple farm. 'nough said.
Also poor Marshall or pig only build dairy farms...
